packages/codegen-ui-react/lib/forms/form-renderer-helper.ts

```diff
@@ -55,6 +55,8 @@ import {
   setStateExpression,
 } from './form-state';
 
+export const shouldWrapInArrayField = (config: FieldConfigMetadata) => config.isArray || config.relationship;
```

Comment:
nit: I wonder if we should start splitting up functionality into separate functions for better readability (ex. relationship helpers into their own files)

Reply:
I agree this form-renderer-helper file is getting unwieldy. Will take note to split this up in a separate PR.